lib/java/com/google/android/material/textfield/res-public/values/public.xml

```diff
@@ -35,6 +35,7 @@
   <public name="boxPaddingBottom" type="attr"/>
   <public name="boxStrokeColor" type="attr"/>
   <public name="boxStrokeWidth" type="attr"/>
+  <public name="defaultBoxStrokeColor" type="attr"/>
```

Comment:
"Default" is a little confusing in an attribute, since it's not clear what the attribute change - does it change the behavior of the default? The color of the box stroke that's used unless boxStrokeColor is set?

This was sort of ok as an internal name, but ideally we use something other than default for the public facing attribute. It would be nice to use boxStrokeColor, but that already exists as the focusedBoxStrokeColor. Rather than overhauling all of the box stroke attributes, how about naming this something like "unfocusedBoxStrokeColor" or "unfocusedBoxStrokeColor", or a better name that implies that it's the color that's used for the non-focused state?

Reply:
I was with some free time so I tried the change...placed it in another branch while waiting a feedback from this pr... https://github.com/BugsBunnyBR/material-components-android/tree/color-state-list

Reply:
Any news about this PR, guys? It seems reasonable to me that the attribute `boxStrokeColor` could receive a ColorStateList